<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif CJK SC</family>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(187, 191, 64);
color: rgb(206, 92, 0);</string>
    </property>
    <property name="text">
     <string>Tableau de bord</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>66</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(52, 101, 164);</string>
    </property>
    <property name="text">
     <string>Élève</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(52, 101, 164);</string>
    </property>
    <property name="text">
     <string>Semestre</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>60</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>semestre 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>semestre 2</string>
     </property>
    </item>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>101</y>
      <width>901</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>500</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(245, 121, 0);</string>
    </property>
    <property name="text">
     <string>Résultast de la classe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>470</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(92, 53, 102);</string>
    </property>
    <property name="text">
     <string>Statistique des résultats</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>341</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Inscrire un(e) élève de la classe</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Prénoms:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Sexe:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Matricule:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Date de naissance:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Statut:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Contact parent/tuteur:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enregisterEleve">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>360</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(138, 226, 52);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Enregistrer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="matricule">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="prenom">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateNaissance">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="adresse">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>240</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="statut">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Redoublant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nouveau</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="sexe">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Masculin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Feminin</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="contact">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>320</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="afficherListeEleve">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(237, 212, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Liste des élèves</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>400</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 41, 41);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Supprimer un élève</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>345</width>
      <height>206</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>URW Chancery L</family>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(52, 101, 164);</string>
       </property>
       <property name="text">
        <string>Statistique de la classe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView2"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="afficherTableCompo">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>60</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(143, 89, 2);
font: 75 italic 16pt &quot;URW Chancery L&quot;;</string>
    </property>
    <property name="text">
     <string>Compositions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouterLigneCompo">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>60</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(78, 154, 6);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimerLigneCompo">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>60</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(164, 0, 0);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="afficherMatiere">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>660</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(191, 64, 138);</string>
    </property>
    <property name="text">
     <string>Afficher les matières</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>471</y>
      <width>711</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1296</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
